src/main/resources/archetype-resources/deployment/k8s/branch/all.jsonnet

```diff
@@ -0,0 +1,180 @@
+local vars = std.extVar('rebuyTemplateVariables');
+
+local presets = {
+  production: {
+    replicas: 2,
+    liveTraffic: true,
+    cpuRequests: '100m',
```

Comment:
Memory should also be here. I suggest simply dumping the whole object, like here: https://github.com/rebuy-de/web/blob/d18fe3b3b031edc1190fb1e51cc28a92de19e2ce/deployment/k8s/blue/all.jsonnet#L19-L22